src/mscorlib/src/System/Diagnostics/Eventing/EventPipeEventProvider.cs

```diff
@@ -66,6 +66,12 @@ uint IEventProvider.EventUnregister(long registrationHandle)
                     return 0;
                 }
 
+                if (eventDescriptor.Channel == 11)
```

Comment:
Can you put a comment about the fact that Channel == 11 means 'Using TraceLogging' and also that TraceLogging passed some extra arguments for ETW that we don't need for EventPipe.   Otherwise this code is completely mysterious.

Reply:
Yes.  Just pushed a change for this.  Thank you for the review!